docs/content/getting-started/create-new-project.mdx

```diff
@@ -58,7 +58,7 @@ dagit
 
 The Dagit process automatically uses the file `workspace.yaml` to find your repositories, from which Dagster will load your jobs, schedules, and sensors. To see how you can customize the Dagit process, run `dagit --help`.
 
-3. (Optional) If you want to enable Dagster [Schedules](/concepts/partitions-schedules-sensors/schedules) or [Sensors](/concepts/partitions-schedules-sensors/sensors) for your jobs, start the [Dagster Daemon process](/deployment/dagster-daemon) **in the same folder as your `workspace.yaml` file, but in a different shell or terminal**:
+3. (Optional) If you want to enable Dagster [Schedules](/concepts/partitions-schedules-sensors/schedules) or [Sensors](/concepts/partitions-schedules-sensors/sensors) for your jobs, start the [Dagster Daemon process](/deployment/dagster-daemon) **in the same folder as your `workspace.yaml` file, but in a different shell or terminal**. Make sure `$DAGSTER_HOME` is set to somewhere other than `/tmp`:
```

Comment:
we should probably explain DAGSTER_HOME here too since it hasnt been mentioned yet

something like

"The $DAGSTER_HOME environment variable must be set to a directory for the daemon to work. Note: using directories within `/tmp` may cause issues. See https://docs.dagster.io/deployment/dagster-instance#default-local-behavior for more details"

but better